### DISCUSSION TITLE
RFC on Vue.js Features

### DISCUSSION THREAD

**Discussion by SkyaTura** [prettier/plugin-pug#109] [RFC]
## Request / Idea
The idea of this thread was, initially, to discuss about specificities on sorting attributes within usage of Vue.js framework, as noticed in #22 .

However, there is a whole more we didn't previewed, and I think would be nice to bring subjects/problems that could turn into a feature, specially with Vue 3.x changes.

### Vue 2.x

- [ ] An _opinionated_ best attribute sorting rules for default configuration
- [ ] Possible problems that could happen/fixed with the above feature

### Vue 3.x

- [ ] An _opinionated_ best attribute sorting rules for default configuration
- [ ] Possible problems that could happen/fixed with the above feature
- [ ] Possible incompatibilities with 2.x that **cannot** be extended when sorting attributes

**Comment by bsiguret** [comment]
Heya,

Just linking this issue [#172](https://github.com/prettier/plugin-pug/issues/172). It's about disabling Vue specific ignorable warnings because it's false positive ones and polluting the terminal. I didn't really thing of an other solution other than a specific Vue flag. Which is maybe not the best implementation
